.gitlab-ci.yml

```diff
@@ -84,7 +84,7 @@ variables:
   DATADOG_AGENT_BUILDERS: v2311927-0e261ab
   DATADOG_AGENT_WINBUILDIMAGES: v2348149-ba6640d
   DATADOG_AGENT_ARMBUILDIMAGES: v2287129-4816dc7
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v2336045-3227a35
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v2402855-1447971
```

Comment:
Is there a corresponding pull-request somewhere?

Reply:
yes: https://github.com/DataDog/datadog-agent-buildimages/pull/40
It was to add "curl"

Reply:
OK thank you!